test/functional/data/invalid_txs.py

```diff
@@ -122,7 +123,8 @@ class SizeTooSmall(BadTxTemplate):
     def get_tx(self):
         tx = CTransaction()
         tx.vin.append(self.valid_txin)
-        tx.vout.append(CTxOut(0, CScript([OP_TRUE])))
+        tx.vout.append(CTxOut(0, CScript([OP_RETURN] * (MIN_PADDING - 1))))
+        assert len(tx.serialize()) == MIN_STANDARD_TX_NONWITNESS_SIZE - 1
```

Comment:
Should this be checking `len(tx.serialize_without_witness()) == 64` ? That's the actual case where this has an impact?

Reply:
Added another assert to attempt to make it abundantly clear